kafka-bin-scripts/README.adoc

```diff
@@ -131,6 +131,9 @@ ifdef::qs[]
 The `<client_id>` and `<client_secret>` are the generated credentials for your service account. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots), clicking *View connection information*, and creating the service account.
 endif::[]
 
+Add a group ID for the consumer to be able to join a consumer group.
```

Comment:
It seems a little bit inconsistent to call out the consumer group ID by name, but not the other properties. Maybe it would be better to revise the step to say:

> 3. In the app-services.properties file, set the Kafka instance client credentials and consumer group ID. Replace the values with your own server and credential information.
>
> All consumers with the same group ID belong to the consumer group with that ID.